lib/src/query_component.dart

```diff
@@ -18,31 +19,51 @@ import 'util.dart';
     ClassProvider(ExcelDart),
     FORM_PROVIDERS,
   ],
-  directives: [coreDirectives, formDirectives],
+  directives: [bsAccordionDirectives, BsInput, coreDirectives, formDirectives],
 )
 class QueryComponent implements OnInit {
-  final populateButtonName = 'populate';
-  final underpacingCheckBoxName = 'Highlight underpacing insertion orders';
+  // Names used in css.
+  static const requestSectionTitle = 'Request Parameter';
+  static const advertiserParameterName = 'advertiserId';
+  static const mediaPlanParameterName = 'campaignId';
+  static const numberOnlyPattern = '^[0-9]*\$';
+  static const insertionOrderParameterName = 'insertionOrderId';
+  static const underpacingCheckBoxName =
+      'Highlight underpacing insertion orders';
+  static const populateButtonName = 'populate';
+  static const parameterAlertText = 'Missing required parameter values.';
+  static bool showAlert = false;
+  static bool showSpinner = false;
+
   final QueryService _queryService;
   final ExcelDart _excel;
 
   // Default selection is byAdvertiser.
-  QueryType queryType = QueryType.byAdvertiser;
-  List<QueryType> queryTypeChoices = QueryType.values;
+  static QueryType queryType = QueryType.byAdvertiser;
```

Comment:
actually think this is more appropriate as an instance type. this is the currently selected query type right?